conftest.py

```diff
@@ -401,10 +402,10 @@ def pytest_configure(config):
     if_reportportal =config.getoption('--reportportal')
 
     try:
-        config._inicache["rp_uuid"]="c896cbb7-d8f0-4cd0-a0f7-18366057af21"
-        config._inicache["rp_endpoint"]="http://18.221.13.146:8080"
+        config._inicache["rp_uuid"] = report_portol_conf.report_portal_uuid
+        config._inicache["rp_endpoint"]= report_portol_conf.report_portal_endpoint
         config._inicache["rp_project"]="default_personal"
-        config._inicache["rp_launch"]="default_TEST_EXAMPLE"
+        config._inicache["rp_launch"]="Qxf2 Page object model launch"
```

Comment:
This and the rp_project values should come from the conf.

Reply:
Fixed it